### PR TITLE
fix(l10n): add folded line indicator translations from vim

### LINKED ISSUE #19124: The "lines" indicator when folded is not translated
### Neovim version (nvim -v)

v0.8.0-dev+500-gf3c8f3e5d

### Vim (not Nvim) behaves the same?

no, vim 8.2.5126

### Operating system/version

Linux 5.10.102.1-microsoft-standard-WSL2 Ubuntu 22.04

### Terminal name/version

GNOME Terminal 3.44.0 using VTE 0.68.0 +BIDI +GNUTLS +ICU +SYSTEMD

### $TERM environment variable

xterm-256color

### Installation

build from repo

### How to reproduce the issue

min.vim
```vim
" {{{
set nocp
set foldmethod=marker
```
nvim --clean -u min.vim min.vim

### Expected behavior

"lines" is translated into "行" in vim
image

### Actual behavior

"lines" is not translated in neovim
image

### PR DESCRIPTION
**Pull request by tomial**
Fix #19124

Added translations from vim

### CODE REVIEW

**Comment by zeertzjq** [comment]
It seems a `Plural-Forms` header is needed: http://www.gnu.org/software/gettext/manual/html_node/Translating-plural-forms.html

**Comment by tomial** [comment]
Thanks, I 've added those missing headers

File: src/nvim/po/af.po [hunk 1]
```diff
@@ -646,10 +647,10 @@ msgstr "&Ok"
 #~ msgstr ""
 
 #, fuzzy, c-format
```
Comment (zeertzjq) [inline src/nvim/po/af.po:649]: It seems `fuzzy` needs to be removed for this to work
```suggestion
#, c-format
```

File: src/nvim/po/fi.po [hunk 2]
```diff
@@ -842,8 +842,10 @@ msgid "filter() argument"
 msgstr "filter()-argumentti"
 
 #, fuzzy, c-format
```
Comment (zeertzjq) [inline src/nvim/po/fi.po:844]: ditto
```suggestion
#, c-format
```

**Pull request merged**